_data/project-managers.yml

```diff
@@ -1,105 +1,96 @@
-    photo: Rafael_Ostrea.JPG
--
-    timestamp: 9/7/2019 10:12:25
-    name: Lizhi (Gary) Yang
-    project: PieSens
-    description: At PieSens, we design, build and maintain actuator controls and sensors for the robot ranging from motor control boards to distance sensor boards. Skills we teach include embedded programming, PCB design (Eagle) and others.
-    bio: I am a sophomore in EECS interested in Robotics and CS in general. Feel free to shoot me any questions on Facebook @GaryYang2000 :)
-    photo: Gary_Yang.jpg
--
-    timestamp: 9/7/2019 14:09:52
-    name: Kaitlyn Chan
-    project: PiE Sens
-    description: PiE Sens is one of the two projects in PiE Electrical. Members of the team go through an iterative design process to put together our electrical system. The staff goes through a brainstorming phase to collect ideas on what sensors are required for the game. Then, the sensors are prototyped and then translated into a PCB (Printed Circuit Board). We teach the staff how to use industry level PCB software to route our boards before having our sponsors send us the completed boards. We then manufacture the boards in house to test and revise.
-    bio: I am currently a sophomore in EECS. I like robots, cats, and anime. Feel free to email me at kaitlynjchan@berkeley.edu with any questions! :)
-    photo: Kaitlyn_Chan.JPG
--
-    timestamp: 9/7/2019 15:01:43
-    name: Karen Darken
+    name: Jose Valera
+    project: CPR
+    description: CPR performs data analytics, networks with industry professionals, and creates grants for resources and funding
+    bio: I am from Los Angeles and love to watch Star Wars movies and collect coins during my free time.
+    photo: Jose.JPG
+-
+    name: Hailey Jang
     project: Dawn
-    description: Dawn is an application that allows students to program and monitor their robots.
-    bio: 'Hi, I''m Karen! I''m a 4th year CS major. I enjoy breakfast, dancing, and looking at other people''s dogs. '
-    photo: Karen_Darken.jpg
+    description: Dawn is a front-end desktop application that allows high school students to program their robots as well as view their statuses.
+    bio: ''
+    photo: Hailey.png
 -
-    timestamp: 9/7/2019 15:02:01
-    name: Matthew Guo
+    name: Lance Jackson
     project: Dawn
     description: Dawn is a front-end desktop application that allows high school students to program their robots as well as view their statuses.
-    bio: 'Hi there! My name’s Matthew, and I’m a second year CS major from the Bay Area. When I’m not fostering Dawn''s culture, I like to spend my time drawing, making video games, and perfecting my (really bad) New Zealand accent. Fun fact about me: I can lick my elbow.'
-    photo: Matthew_Guo.jpg
+    bio: ``
```

Comment:
jekyll 3.8.6 | Error:  (/home/travis/build/pioneers/website/_data/project-managers.yml): found character that cannot start any token while scanning for the next token at line 29 column 10